ci-operator/step-registry/ipi/deprovision/deprovision/ipi-deprovision-deprovision-commands.sh

```diff
@@ -11,4 +11,4 @@ export AZURE_AUTH_LOCATION=$cluster_profile/osServicePrincipal.json
 echo "Deprovisioning cluster ..."
 cp -ar "${SHARED_DIR}" /tmp/installer
 openshift-install --dir /tmp/installer destroy cluster
-cp /tmp/installer/.openshift_install.log "${ARTIFACT_DIR}/"
+cp /tmp/installer/.openshift_install.log "${ARTIFACT_DIR}/.openshift_install.deprovision.log"
```

Comment:
I thought ARTIFACT_DIR was per-step?  So deprovision steps are already using a different ARTIFACT_DIR than the install steps, right?